docker-test-env/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.9"
```

Comment:
Lots happening throughout this file, few comments would go a long way

Reply:
I created L2-858 to track this.